### PR TITLE
Requested Croatia and Hungary position changes + minor fixes

### PR DESCRIPTION
**Pull request by IhateTrains**
Do you know how many positions have been changed in these areas?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -223,3 +223,5 @@ pip-log.txt
 #############
 
 *.exe
+WTWSMS/events/WtWSMS_story_events.txt
+WTWSMS/events/WtWSMS_imperial_legacy_events.txt
```
Comment (loup99) [inline .gitignore]: This shouldn't be in.

File: WTWSMS/decisions/mnm_society_decisions.txt [hunk 2]
```diff
@@ -1022,7 +1022,7 @@ targeted_decisions = {
 		only_playable = yes
 		
 		ai_target_filter = realm
-		ai_target_filter = rivals
+		#ai_target_filter = rivals # I don't know which one to remove, only one is allowed
```
Comment (loup99) [inline WTWSMS/decisions/mnm_society_decisions.txt]: It works, keep both.

File: WTWSMS/events/WtWSMS_imperial_legacy_events.txt [hunk 3]
```diff
@@ -1625,4 +1625,4 @@ narrative_event = {
 		
 		prestige = -100
 	}
-}
+}
```
Comment (loup99) [inline WTWSMS/events/WtWSMS_imperial_legacy_events.txt]: ❔

File: WTWSMS/events/WtWSMS_story_events.txt [hunk 4]
```diff
@@ -3577,4 +3577,4 @@ narrative_event = {
 	option = { 
 		name = historical_wtwsms.6.a
 	}
-}
+}
```
Comment (loup99) [inline WTWSMS/events/WtWSMS_story_events.txt]: ❓

File: WTWSMS/events/WtWSMS_imperial_legacy_events.txt (hunk 3 above)
Comment (IhateTrains) [inline WTWSMS/events/WtWSMS_imperial_legacy_events.txt]: I can't see the difference o.O

File: WTWSMS/decisions/mnm_society_decisions.txt (hunk 2 above)
Comment (IhateTrains) [inline WTWSMS/decisions/mnm_society_decisions.txt]: OK

**Comment by loup99** [comment]
To see the map changes, revert your Steam to 2.6.3, copy the relevant map files and then go back to 2.7.2 and compare.

File: WTWSMS/events/WtWSMS_imperial_legacy_events.txt (hunk 3 above)
Comment (loup99) [inline WTWSMS/events/WtWSMS_imperial_legacy_events.txt]: I don't get it either.

**Comment by IhateTrains** [comment]
I didn't think of that. Thanks :)

**Comment by IhateTrains** [comment]
It's ready :)

**Pull request merged**